src/test/resources/catalog-with-dataset-with-qualified-attributions.ttl

```diff
@@ -0,0 +1,15 @@
+@prefix rdf:   <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix dcat:  <http://www.w3.org/ns/dcat#> .
+@prefix prov:  <http://www.w3.org/ns/prov#> .
+
+<http://catalog/1/dataset/1>
+    a                          dcat:Dataset ;
+    prov:qualifiedAttribution  [ a           prov:Attribution ;
+                                 prov:agent  <https://data.brreg.no/enhetsregisteret/api/enheter/123456789>
```

Comment:
Vil gjerne at du legger property:
```
dcat:hadRole <http://registry.it.csiro.au/def/isotc211/CI_RoleCode/bladibladibla>
```
der du henter relevant rolle frå https://standards.iso.org/iso/19115/resources/Codelists/gml/CI_RoleCode.xml
Tenker, som du foreslo, at  dcat:hadRole <http://registry.it.csiro.au/def/isotc211/CI_RoleCode/contributor> er den beste her.

Reply:
Done